src/mesh/SimRadio.cpp

```diff
@@ -0,0 +1,250 @@
+#include "SimRadio.h"
```

Comment:
Since these (h/cpp) are only for Portduino, please move them to src/platform/portduino. Else all other targets will compile (but not link) them